pkgs/by-name/ht/htcondor/package.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libuuid
+, expat
+, curl
+, pcre2
+, sqlite
+, python3
+, boost
+, libxml2
+, libvirt
+, munge
+, voms
+, perl
+, scitoken-cpp
+, openssl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "htcondor";
+  version = "23.3.0";
+
+  src = fetchFromGitHub {
+    owner = "htcondor";
+    repo = "htcondor";
+
+    rev = "v23.3.0";
+    hash = "sha256-Ew9leVpvEndiRkOnhx2fLClrNW1bC5djcJEBsve6eIk=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [
+    libuuid
+    expat
+    openssl
+    curl
+    pcre2
+    sqlite
+    python3
+    boost
+    libxml2
+    libvirt
+    munge
+    voms
+    perl
+    scitoken-cpp
+  ];
+
+
+  cmakeFlags = [ "-DSYSTEM_NAME=NixOS" "-DWITH_PYTHON_BINDINGS=false" ];
```

Comment:
```suggestion
  cmakeFlags = [
    "-DSYSTEM_NAME=NixOS"
    "-DNO_PHONE_HOME=ON"
    "-DWITH_PYTHON_BINDINGS=false"
  ];
```

Reply:
Why specificity the NO_PHONE_HOME, i don't see this in any of the cmake files of this project nor in the cmake docs it self? I do see it in the git history of the project but this phone home feature has been removed a while ago.

Reply:
You're right, I've looked at outdated specfile. This is not a thing since a while now:
https://github.com/htcondor/htcondor/commit/733bec5ff33988bf91c744666ac89040e156f2af